code/modules/uplink/uplink_items/explosive.dm

```diff
@@ -0,0 +1,103 @@
+// File ordered based on progression.
+
+/datum/uplink_category/explosives
+	name = "Explosives"
+	weight = 6
+
+/datum/uplink_item/explosives
+	category = /datum/uplink_category/explosives
+
+// Low progression cost
+/datum/uplink_item/explosives/soap_clusterbang
+	name = "Slipocalypse Clusterbang"
+	progression_minimum = 10 MINUTES
+	desc = "A traditional clusterbang grenade with a payload consisting entirely of Syndicate soap. Useful in any scenario!"
+	item = /obj/item/grenade/clusterbuster/soap
+	cost = 3
+
+// Medium progression cost
+
+/datum/uplink_item/explosives/c4
```

Comment:
As I said with the destroy heirloom objective, the c4 brick is one of the few ways you can destroy an item quickly. I think this has uses beyond just being a weapon, and while people do use it as a weapon, it also has plenty of practical utility as well. I usually save a few TC just for c4 blocks. I can agree on a 20 minute wait time on the c4 bag, but not for the normal blocks. Maybe put them at 10 minutes. You'll know if you need them.